deploy/src/main/resources/kubernetes.env

```diff
@@ -0,0 +1,29 @@
+# The following properties MUST be configured before the Job Service can be deployed.
```

Comment:
Why do all of these have to be configured?  Can we not have sensible defaults for most of them and/or aren't a lot of them internal?

Reply:
I have no idea how to set them as defaults in the templates.

Reply:
It feels like it is too much to ask people to fill out for a quick-start, esp. without a lot of guidance w.r.t. what the values should be.

Reply:
I appreciate that but I do not know how to set them as defaults.

Reply:
@rado-straka Do you know how to do this?

Reply:
I use services in kubernetes, so for example rabbitmq can be called by its name and using port which are defaults in code:
logs from familyhashing:
```
 INFO  [2019-01-22 15:56:47,575] net.jodah.lyra.internal.ConnectionHandler: Creating connection cxn-1 to [rabbitmq:5672]
INFO  [2019-01-22 15:56:47,665] net.jodah.lyra.internal.ConnectionHandler: Created connection cxn-1 to amqp://10.101.108.140:5672/ 
```
servcie definition is:
```
apiVersion: v1
kind: Service
metadata:
  labels:
    io.kompose.service: rabbitmq
  name: rabbitmq
spec:
  ports:
  - name: "rabbitmq-ctrl-port"
    port: 15672
    #if targetPort is not set then targetPort==port 
    #targetPort: 15672
  - name: "rabbitmq-node-port"
    port: 5672
    #targetPort: 5672
  selector:
    io.kompose.service: rabbitmq
```
so this service is called by its name "rabbitmq" and pointing to the rabbitmq pod what is done by selector section which must be the same as the label in rabbitmq pod.
username and password for rabbitmq also should work with defaults in code.

For other env variables it would need to check what are defaults for some of them, and if they match with setup of other services.